### DISCUSSION TITLE
Some details about 《Definition of the Floating System for Phase IV of OC3》

### DISCUSSION THREAD

**Discussion by sjihf12** [OpenFAST/openfast#1701] [Q&A] [answered]
Dear Professors
I recently read 《Definition of the Floating System for Phase IV of OC3》, the author is Professor J.Jonkman. I have some doubts, and I hope you can help me solve them.
Q1: The mass of the floating platform is 7466330kg. The buoyancy force in undisplaced position is 80708100N. 7466330kg*9.8N/kg=73170034, not equal to 80708100N. The buoyancy and gravity of the platform are not balanced, this puzzles me. Can you kindly solve this confusion for me, thank you!
Q2:Can you tell me in detail how to calculate the "Platform Roll Inertia about CM" and "Platform Yaw Inertia about  Platform Centerline"，especially the tower, the Nacelle and the blades. Thank you!

**Comment by jjonkman** [comment, marked as answer]
Dear @sjihf12,

Regarding (1), your calculations neglect the weight of the supported wind turbine (rotor, drivetrain, nacelle, and tower) and vertical pretension of the mooring system; when these are accounted for, all static vertical loads should balance.

Regarding (2), the following topic on our forum where additional inertia data of the NREL 5-MW baseline wind turbine atop the OC3-Hywind spar are provided is likely of interest to you: https://forums.nrel.gov/t/inertial-moments-of-oc3-hywind-components/610.

Best regards,

**Reply by sjihf12** [reply]
Thank you!
